dracut/99setup-root/initrd-setup-root

```diff
@@ -82,9 +82,13 @@ done
 /usr/sbin/flatcar-tmpfiles /sysroot
 
 # Initialize base filesystem without /etc
+# Don't run all tmpfiles rules but only the essential ones
+# required to let systemd do its work correctly. The rest
+# will be done later as usual.
 systemd-tmpfiles --root=/sysroot --create \
     baselayout.conf baselayout-usr.conf \
-    baselayout-home.conf
+    baselayout-home.conf base_image_var.conf \
+    var.conf
```

Comment:
Where the `base_image_var.conf` and `var.conf` come from?

Reply:
All these files are under `/sysroot/usr/lib/tmpfiles.d/`

Reply:
Sorry for the confusion. I meant that I know that baselayout-home.conf comes from our baselayout project, but I don't know from which project base_image_var.conf and var.conf come from. I couldn't find them in systemd, init or baselayout. That's why I'm asking.

Reply:
`base_image_var.conf` is generated based on package contents, `var.conf` comes from systemd.

Ok, sounds like we should not add `var.conf` directly but only copy the wanted line somehow